language/German/strings.po

```diff
@@ -3970,575 +4031,575 @@ msgstr ""
 #, c-format
 msgctxt "input-name"
 msgid "%p Select"
-msgstr ""
+msgstr "%p Auswahl"
```

Comment:
Is the "Select" key not also usually called "Select" in German?

Reply:
Don't know.

Reply:
I think it’s usually just called “Select” in German in Nintendo and Microsoft console documentation anyway.  Probably best to stick with that.

Reply:
Ok, Thanks. Fixed.